kratos/python_scripts/python_linear_solver_factory.py

```diff
@@ -83,8 +82,7 @@ def __CheckIfSolverTypeIsDeprecated(config):
         "SuperLUSolver"          : "super_lu",
         "SuperLUIterativeSolver" : "super_lu_iterative",
         "PastixSolver"           : "pastix",
-        "PastixComplexSolver"    : "pastix_complex",
-        "FEASTSolver"            : "feast"
+        "PastixComplexSolver"    : "pastix_complex"
```

Comment:
can you please revert the changes in this file?
I saw that it needs some cleanup, which I would like to do in a separate PR

Reply:
can you please do this, then I think it is ready

Reply:
sorry, missed that